ethereum_history_api/oracles/src/script/noir_fixtures/storage_proof.ts

```diff
@@ -0,0 +1,50 @@
+import { Hash } from 'viem';
+import { encodeHexString } from '../../noir/noir_js/encode.js';
+import { encodeProof } from '../../noir/oracles/accountOracles.js';
+import { encodeBytes32 } from '../../noir/oracles/encode.js';
+
+export function createStorageProofFixture(
+  storageRoot: Hash,
+  storageProofs: { key: Hash; proof: Hash[]; value: bigint }[]
```

Comment:
Let's extract the StorageProof type

Reply:
Ok, extracted. In fact this type exist in viem library but it is not exported.